galaxy_importer/galaxy-importer.cfg

```diff
@@ -0,0 +1,4 @@
+[galaxy-importer]
+LOG_LEVEL_MAIN = INFO
+RUN_ANSIBLE_TEST = False
```

Comment:
Lets make it true, and check whether galaxy_import itself is running the test sanity.
If yes, then we can remove explicit call to `ansible-test sanity` from Jenkins.

Reply:
Set to True. But it doesn't run ansible sanity test explicitly when I used this command
`python3 -m galaxy_importer.main --git-clone-path ./ --output-path ../../../ansible_collections_out`